core/services/src/main/java/org/exoplatform/services/attachments/utils/EntityBuilder.java

```diff
@@ -86,37 +86,37 @@ public static final Attachment fromAttachmentNode(RepositoryService repositorySe
     } catch (ItemNotFoundException e) {
       throw new ObjectNotFoundException("Node with id " + attachmentId + " wasn't found");
     }
-
+    Node originalDocumentNode = attachmentNode;
     if (linkManager.isLink(attachmentNode)) {
-      attachmentNode = linkManager.getTarget(attachmentNode);
-      if (attachmentNode == null) {
+      originalDocumentNode = linkManager.getTarget(attachmentNode);
+      if (originalDocumentNode == null) {
         throw new ObjectNotFoundException("Target Node with of symlink " + attachmentId + " wasn't found");
       }
     }
 
     Attachment attachment = new Attachment();
-    attachment.setId(attachmentNode.getUUID());
-    String attachmentsTitle = getStringProperty(attachmentNode, "exo:title");
+    attachment.setId(originalDocumentNode.getUUID());
```

Comment:
here for Attachment they use the UUID of the Node

Reply:
Before this [fix commit](https://github.com/exoplatform/ecms/commit/e929b5b8ff82ab8b98af5ca13cee43316574551e) there were some exceptions related to the getNodeByUUID on a symlink since it is not referenceable, that's why we have changed in all related location getNodeByUUID  by getNodeByIdentitfier. 
The problem now is that the displayed breacrumb corresponds to the original document not the symlink, so in the case of a user previewing the shared document (symlink) without permissions on the original document, the breacrumb will not be displayed correcly. IMO, in case of a symlink we should we should display its breacrumb and not the original document one.